Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMultiView.css

```diff
@@ -7,81 +7,63 @@ div.tabbedMultiView
   position: relative;
   margin: 0;
   padding: 0;
-  background-color: white;
 }
 
 div.tabbedMultiView > div.wrapper
 {
   position: absolute;
-  background-color: white;
   top: 0;
   bottom: 0;
   left: 0;
   right: 0;
-  width: auto;
-  height: auto;
+  padding-top: var(--remotion-themed-spacing-double);
+  padding-right: var(--remotion-themed-spacing-double);
+  padding-left: var(--remotion-themed-spacing-double);
+
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: max-content max-content auto max-content;
+  grid-template-areas: "topControls" "tabStrip" "view" "bottomControls";
 }
 
 div.tabbedMultiViewActiveView
 {
-  height: 80%;
-  width: inherit;
+  grid-area: view;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
 }
 
 div.tabbedMultiViewActiveView > div.contentBorder
 {
-  position: relative;
-  background-color: white;
-  margin: 0; 
-  padding: 0.5em;
+  padding: var(--remotion-themed-spacing-half);
   overflow: auto;
-  height: 60%;
+  margin: 1px; /* this small margin is needed to correctly display the outline in FF */
 }
 
-div.tabbedMultiViewActiveView > span:focus + div.contentBorder 
+div.tabbedMultiViewActiveView > div.contentBorder:focus
 {
   outline: dotted black 1px;
   outline: auto 5px -webkit-focus-ring-color;
 }
 
-div.tabbedMultiViewActiveView > div.contentBorder > div.content
-{
-  position: relative;
-  left: 0;
-  top: 0;
-  width: 100%;
-  height: 100%;
-}
-
-/* Applied to the top section of the multi view. Height cannot be provided in percent. */
+/* Applied to the top section of the multi view. */
 div.tabbedMultiViewTopControls
 {
-  overflow: hidden;
-  background-color: White;
-}
-
-div.tabbedMultiViewTopControls.empty
-{
-  height: 1em;
-  background-color: White;
+  grid-area: topControls;
 }
 
-/* Applied to the bottom section of the multi view. Height cannot be provided in percent. */
+/* Applied to the bottom section of the multi view. */
 div.tabbedMultiViewBottomControls
 {
-  position: absolute;
-  overflow: hidden;
-  background-color: White;
-  margin-top: 0.5em;
-  left: 0.5em;
-  right: 0.5em;
-  bottom: 0.5em;
+  grid-area: bottomControls;
+  padding: var(--remotion-themed-spacing-half);
 }
 
-div.tabbedMultiViewBottomControls.empty
+/* Applied to the tab strip section of the multi view. */
+div.tabbedMultiViewTabStrip
 {
-  height: 1em;
-  background-color: White;
+  grid-area: tabStrip;
 }
 
 /* Shared in other CSS files */
```

Comment:
When you drop the single usage of screenReaderText in the rendering, this style can go away, too. Same in NovGray css.